packages/ra-ui-materialui/src/detail/CreateView.tsx

```diff
@@ -22,10 +22,7 @@ export const CreateView = (props: CreateViewProps) => {
     const { resource, defaultTitle } = useCreateContext();
 
     return (
-        <Root
-            className={clsx('create-page', className)}
-            {...sanitizeRestProps(rest)}
```

Comment:
I'm surprised this works, since CreateViewProps is declared as CreateProps (which is a mistake, as there are some Create props that CreateView will never get injected)

Reply:
And yet it does without any warning at runtime in all demos